setup.py

```diff
@@ -552,6 +552,8 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
 # DEPENDENCIES_EPOCH_NUMBER in the Dockerfile.ci
 #####################################################################################################
 INSTALL_REQUIREMENTS = [
+    'WTforms<2.3.0',
```

Comment:
Why not change this to `wtforms<2.3.0` . `pip` is case-insensitive

Reply:
https://www.python.org/dev/peps/pep-0426/#name

Reply:
Ah dod not know that :). You learn every day ! Will change

Reply:
Scary:

All comparisons of distribution names MUST be case insensitive, and MUST consider **hyphens and underscores** to be equivalent. :scream: